torch/csrc/jit/mobile/import.cpp

```diff
@@ -140,16 +158,19 @@ void parseMethods(
     }
 
     std::unordered_set<std::string> unsupported_op_names;
-    for (const auto& op : ops_list) {
-      auto op_item = op.toTuple()->elements();
+    for (size_t i = 0; i < ops_list.size(); ++i) {
+      auto op_item = ops_list[i].toTuple()->elements();
       TORCH_CHECK(
           op_item.size() == 2,
           "There should be two parts in an operator name.");
+      const std::string& module_debug_info = (hasDebugInfo) ?
+          module_debug_info_list[i].toString()->string() : "";
       auto op_found = function->append_operator(
           op_item[0].toString()->string(), op_item[1].toString()->string());
+      function->append_module_info(module_debug_info);
       if (!op_found) {
         unsupported_op_names.emplace(operator_str(
-            op_item[0].toString()->string(), op_item[1].toString()->string()));
+            op_item[0].toString()->string(), op_item[1].toString()->string(), module_debug_info));
```

Comment:
Since this function is used only for unsupported op names, let's not add module info to that. The error message out of this becomes too long and unclear. unsupported_op_names is mainly used to indicate what ops are not supported. No need to add debug info there.

Reply:
That's good to know. I'll fix that.